apiary.apib

```diff
@@ -3342,6 +3344,10 @@ Get a list of project milestones.
                 + depends_on (object, nullable)
                     + type: `milestone` (string)
                     + id: `0488d792-ba9b-059f-bd57-bea75d3f4f4e` (string)
+                + dependency_for (array)
```

Comment:
Is it really necessary to have the relationship modelled in two directions?
Where will we use this?

Reply:
Yes. The frontend will tell our users if their changes to due dates (or dependencies) will mess up the project's flow. That goes for both the "parent" and "children". An alternative would be to create a filter, but that would make it all more complex, and would mean more API calls